runtime/mainnet/src/config/xcm.rs

```diff
@@ -170,7 +163,7 @@ impl xcm_executor::Config for XcmConfig {
 	type Barrier = Barrier;
 	type CallDispatcher = RuntimeCall;
 	type FeeManager = XcmFeeManagerFromComponents<
-		WaivedLocations,
+		(),
```

Comment:
Can we add a simple comment saying: "No locations have waived fees".

Reply:
[65e07a7](https://github.com/r0gue-io/pop-node/pull/456/commits/65e07a783b350e30884bec9eebdf73cab744b9d5)